installer/tue-install-impl.bash

```diff
@@ -1229,6 +1232,14 @@ stamp=$(date_stamp)
 INSTALL_DETAILS_FILE=/tmp/tue-get-details-$stamp
 touch "$INSTALL_DETAILS_FILE"
 
+for branch in $BRANCH
+do
+    tue-install-debug "Parsed branch $branch"
+    # Switch tue-env immediately to this branch:
+    _try_branch "$TUE_DIR" "$branch"
+    [[ -n $_try_branch_res ]] && tue-install-info "[tue-env] $_try_branch_res"
```

Comment:
What `tue-env` doing here? Both the comment and the code

Reply:
@Timple why do we check out `tue-env` to a different version? In the current setup `tue-env` is only pulled, never checked out. Not even in CI.